tests/consensus_tests/test_named_vector_crud.py

```
@@ -0,0 +1,299 @@
+def wait_collection_vector_config(peer_url, collection_name, vector_name, expected_size):
+    """Wait until a peer's collection config contains the given vector with the expected size."""
+    def check():
+        vectors = get_collection_vectors_config(peer_url, collection_name)
+        return vector_name in vectors and vectors[vector_name].get("size") == expected_size
+
+    wait_for(check)
+
+
+def get_optimizer_status(peer_url, collection_name):
+    """Get optimizer status from collection info."""
+    info = get_collection_info(peer_url, collection_name)
```

Ruff (0.15.9)

[error] `get_collection_info` may be undefined, or defined from star imports

(F405)